src/index.js

```diff
@@ -0,0 +1,115 @@
+import getWeatherData from './modules/requester';
+import './styles.css';
+
+const displayValues = (response) => {
+  console.log(response);
```

Comment:
- [ ] Please remove the console logs from your application, they are ok for debugging purposes but should be removed before submtiting for review or deploying the code as they make the code messy and overflow the browser terminal.